internals/helpers/git-helper.js

```diff
@@ -45,7 +46,11 @@ async function checkIfRepositoryIsCleanable() {
  * @returns {Promise<any>}
  */
 async function removeGitRepository() {
-  return exec('rm -Rf .git')
+  if (os.platform() === 'win32') {
```

Comment:
Could you do this change in a separate PR ? It's would be easier for me :) 
Why don't just do a rimraf of the `.git` folder ? It would avoid doing an if / else statement

Reply:
I can change it to a seperate PR for sure.

I can also change it to the rimraf it felt a bit over kill for a single command. Entirely your decision. Let me know

Reply:
The advantage of using `rimraf` is that we can be sure that it will work on any platform :)

Reply:
Seems like that is the plan. I will get it back today.

Reply:
Thank you so much !